cli/azd/extensions/azure.ai.agents/internal/pkg/agents/agent_api/operations.go

```diff
@@ -549,10 +549,16 @@ func (c *AgentClient) DeleteAgentEventHandler(ctx context.Context, agentName, ev
 // Container Operations
 
 // StartAgentContainer starts a container for a specific version of an agent
-func (c *AgentClient) StartAgentContainer(ctx context.Context, agentName, agentVersion string, apiVersion string) (*AcceptedAgentContainerOperation, error) {
+func (c *AgentClient) StartAgentContainer(ctx context.Context, agentName, agentVersion string, minReplicas *int32, maxReplicas *int32, apiVersion string) (*AcceptedAgentContainerOperation, error) {
```

Comment:
nit. Move all the optional args into a `StartAgentContainerOptions` struct which can continue to grow w/o you adding more and more args to the function signature.